### ISSUE TITLE
Port forwarding - "Taking it global": @font-face rendering and CSS not loading

### ISSUE THREAD

**Issue by 1-800-jono** [bdkjones/CodeKit2#526]
Hi - I've been trying to use the Port forwarding/"taking it global" functionality but having a few issues with everyone I've asked to check the project out at [ip here]:5757

1 ) Not sure if this is all the time but today @font-face fonts were not rendering - got a default font instead.

2 ) For some people the site loads without CSS 

3 ) Everyone sees the big orange banner saying it's trying to connect for auto-refresh (You already tweet it me you were gonna fix this for next update)

Your app is awesome and I love it so don't get mad :)

**Comment by bdkjones** [comment]
Yea, this is one of those things that’s not _really_ supported because it works when it works and fails when it fails and which camp it falls into is determined primarily by the amount of lag between each person viewing the remote connection. 

What’s the address for the remote server?

**Comment by 1-800-jono** [comment]
Ok - I noticed that problems 1 and 2 weren't happening on my other Wordpress projects. Tried to debug - long story short I re-installed Wordpress and problems 1 and 2 were gone.

Maybe installing Wordpress while using Codekit messes something or it was something else.

So only problem is the big orange banner - which you already said were going to fix so issue closed I guess.

Thanks for responding and this cool app.
